styles/components/app-nav.css

```diff
@@ -0,0 +1,52 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.app-grid {
+  display: grid;
+  grid-template-rows: 48px 1fr 48px;
+}
+
+.nav-container {
```

Comment:
same her and below, starting with: app-nav 😊

Reply:
same solution i think - i have too much classes starting with quiz according to the title of the project